src/OpenTelemetry.Contrib.Extensions.AWSXRay/Resources/AWSECSResourceDetector.cs

```diff
@@ -34,18 +38,30 @@ public class AWSECSResourceDetector : IResourceDetector
     /// <returns>List of key-value pairs of resource attributes.</returns>
     public IEnumerable<KeyValuePair<string, object>> Detect()
```

Comment:
There is some changes in public API in this package.
I think it will be great to change the implementation from `OpenTelemetry.Contrib.Extensions.AWSXRay.Resources.IResourceDetector` to `OpenTelemetry.Resources.IResourceDetector` and remove the old one interface.

It is candidate do separate PR.

Reply:
That is what I am doing now, but it is across the entire .csproj

Reply:
That is what I am doing now, and it cannot be a follow-up PR (unless I am mistaken) if we want to follow the train of the last major sdk release. Unfortunately, it is across the entire .csproj

Reply:
Work started in c213eadc2bec3a67c433ccc83a7584d178d98001